src/components/form/templates/array-field.ts

```diff
@@ -1 +1,63 @@
-export const ArrayFieldTemplate = undefined; // TODO implement array field template
+import React from 'react';
+import { CollapsibleItemTemplate } from './array-field-collapsible-item';
+import { SimpleItemTemplate } from './array-field-simple-item';
+import { renderDescription, renderTitle } from './common';
+import { ArrayFieldItem, ArrayFieldTemplateProps } from './types';
+
+export class ArrayFieldTemplate extends React.Component {
+    private props: ArrayFieldTemplateProps;
+
+    constructor() {
+        super();
+        this.renderItem = this.renderItem.bind(this);
+        this.handleAddClick = this.handleAddClick.bind(this);
+    }
+
+    public render() {
+        return React.createElement(
+            'div',
+            {},
+            renderTitle(this.props.title),
+            renderDescription(this.props.schema.description),
+            this.renderAddButton(),
+            this.props.items.map(this.renderItem)
+        );
+    }
+
+    private renderAddButton() {
+        if (!this.props.canAdd) {
+            return;
+        }
+
+        return React.createElement('limel-button', {
+            outlined: true,
+            label: 'Add item',
```

Comment:
This string should definitely be translated though, and I think it should be provided by the consumer.

Reply:
Yeah, I didn't know what to do about it right now but we need to be able to translate it somehow :+1: